GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/NecromancerHelper.cs

```diff
@@ -123,9 +123,7 @@ internal static class NecromancerHelper
 
     public static bool IsShroudTransform(long id)
     {
-        return _shroudTransform.Contains(id)
-            || ReaperHelper.IsReaperShroudTransform(id)
-            || HarbingerHelper.IsHarbingerShroudTransform(id);
+        return _shroudTransform.Contains(id);
```

Comment:
should be restored, add a public static bool IsDeathShroudTransform for your weapon swap instead